lib/lookup.json

```diff
@@ -405,24 +344,6 @@
     "maintainers": "kriskowal",
     "skip": "win32"
   },
-  "react": {
-    "prefix": "v",
-    "maintainers": ["acdlite", "gaearon", "sophiebits"],
-    "yarn": true,
-    "skip": [">=12", "aix", "s390", "ppc"]
-  },
-  "readable-stream": {
```

Comment:
We're not running it since Node.js 12 https://github.com/nodejs/citgm/commit/813ba66db03acf0af8bf95c585169fbcc163d8f7.

As I said, the best approach IMO would be to open an issue to add this module again.